### DISCUSSION TITLE
[Idea] Improvement of displaying header toolbar on mobile

### DISCUSSION THREAD

**Discussion by ga-devfront** [PrestaShop/PrestaShop#32151] [Ideas]
### The problem

Currently when we only have the help button present in the toolbar of the back office or when we have no button (help_button = false) we still display the button (+) at the bottom right in the mobile version. Which overloads the interface more than anything else.
You will find below the example of the button, (+) while the list of buttons is empty!
![image](https://user-images.githubusercontent.com/52718717/231790187-eb3f36fd-666b-4516-8e37-7cff7eb6fc94.png)

### The solution

Prevent the rendering of the (+) button if the list is empty. As well as the persistence of the help button in the toolbar if it is the only one present.
You can see example bellow.
![image](https://user-images.githubusercontent.com/52718717/231787293-85337498-d5bd-4e40-a13a-9a98dd22c4b0.png)
![image](https://user-images.githubusercontent.com/52718717/231787891-fb5e8390-c1b1-495c-8ce3-2993a2f705e0.png)

### Additional context

I am currently working on a module that needs to display a save bar at the bottom of the page, except on mobile it is superimposed with the (+) button while our toolbar is empty and there is no way to prevent its display.

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [X] I'm willing to provide any wireframes or design assets required for this feature.
- [X] I'm willing to submit a Pull Request that implements this feature.
- [X] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by ElodieOS** [comment]
Hello @ga-devfront!

Thank you for your contribution. This is a very good idea!

You are right, if we don't have an action in the (+) button, we don't need to display it. 

For the help button, in fact, we don't expect to find a help in the (+) button. In my opinion, in mobile, we can put an icon button to save space, especially when we have a large title. 

![image](https://user-images.githubusercontent.com/113457393/231976733-260f835d-3c49-4f27-9bee-3bdb6799b42f.png)

Wdty?

**Reply by ga-devfront** [reply]
done on the PR

**Comment by prestascott** [comment]
@ElodieOS looks good to me!

**Comment by eternoendless** [comment]
Done in https://github.com/PrestaShop/PrestaShop/pull/32152
